src/routes/index.tsx

```diff
@@ -273,28 +261,34 @@ export const router = createBrowserRouter([
       },
       /* Trigger Release plan */
       {
-        path: `workspaces/:${RouterParams.workspaceName}/release/release-plan/trigger/:${RouterParams.releasePlanName}`,
+        path: RELEASEPLAN_TRIGGER_PATH.createPath({
+          workspaceName: RouterParams.workspaceName,
+          releasePlanName: RouterParams.releasePlanName,
+        }),
         loader: releasePlanTriggerLoader,
         errorElement: <RouteErrorBoundry />,
         element: <TriggerReleaseFormPage />,
       },
-      /* Create Release plan */
+      /* Edit Release plan */
       {
-        path: `workspaces/:${RouterParams.workspaceName}/release/release-plan/edit/:${RouterParams.releasePlanName}`,
+        path: RELEASEPLAN_EDIT_PATH.createPath({
```

Comment:
move all of release related routes that are not application scoped to `page-routes/release-service.ts